src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -647,4 +647,8 @@ export class ExerciseAssessmentDashboardComponent implements OnInit, AfterViewIn
             }
         }
     }
+
+    asTextSubmission(submission: Submission): TextSubmission {
```

Comment:
I think we had a discussion where we don't use functions to cast objects because that may slow down on each detection right?

Reply:
Wasn't that for casting that appears many times?